equinix/fabric_connection_schema.go

```diff
@@ -296,8 +296,8 @@ func createConnectionSideAccessPointRes() *schema.Resource {
 			"type": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validation.StringInSlice([]string{"COLO", "VD", "VG", "SP", "IGW", "SUBNET", "GW"}, true),
```

Comment:
Is the "GW" value still supported by the API, or would existing configs that have `type: "GW"` already be broken because the API no longer supports that value?

Reply:
Yes, this has been changed to CLOUD_ROUTER on the API side as well and no longer supports GW.